pipeline/alert.go

```diff
@@ -768,6 +770,11 @@ type AlertaHandler struct {
 	// Default: {{ .Name }}
 	Resource string
 
+	// Alerta event.
+	// Can be a template and has access to the same data as the AlertNode.Details property.
```

Comment:
Can you update this comment since it uses on the Info struct and not the Details?